src/containers/Presentation.js

```diff
@@ -0,0 +1,21 @@
+import Presentation from 'components/Presentation'
+import { 
+    setActiveSlide,
+} from 'actions'
+import { connect } from 'react-redux'
+
+const mapStateToProps = state => {
+    return {
+        slides:      state.presentation.get('slides').toJS(),
```

Comment:
We should avoid making .toJS() calls in mapStateToProps to avoid incurring performance hits